server/handler/base.go

```diff
@@ -146,32 +146,43 @@ func (b *Base) Evaluate(ctx context.Context, installationID int64, trigger commo
 
 	fetchedConfig, err := b.ConfigFetcher.ConfigForPR(ctx, prctx, client)
 	if err != nil {
-		return errors.WithMessage(err, fmt.Sprintf("failed to fetch policy: %s", fetchedConfig))
+		return errors.WithMessage(err, fmt.Sprintf("Failed to fetch configuration: %s", fetchedConfig))
 	}
 
-	return b.EvaluateFetchedConfig(ctx, prctx, client, fetchedConfig, trigger)
+	evaluator, err := b.ValidateFetchedConfig(ctx, prctx, client, fetchedConfig, trigger)
+
+	if evaluator == nil {
```

Comment:
This pattern reads a bit oddly to me. I think I'd prefer to see something like:

```go
if err != nil {
  return err
}
if evaluator == nil {
  return nil
}
```

I think that more clearly communicates that one of the "success" cases is when the function returns a nil evaluator.